templates/telemetry/voice/voice.4.schema.json

```diff
@@ -141,6 +140,21 @@
           "type": "number",
           "minimum": 0
         },
+        "localHour": {
+          "title": "Local hour of action",
+          "description": "The hour, 0-23, in the local timezone when the action was run",
+          "type": "number"
```

Comment:
Somewhat of an implementation detail of how the schemas are transformed into table schemas, but you can use `integer` here so the columns are treated as `INT64` instead of `FLOAT64`.